kse/src/org/kse/gui/dialogs/DVerifyCertificate.java

```diff
@@ -92,46 +108,56 @@ private void initComponents() {
 		jrbChainCheck.setToolTipText(res.getString("DVerifyCertificate.jrbChainCheck.tooltip"));
 
 		ButtonGroup buttonGroup = new ButtonGroup();
-		buttonGroup.add(jrbCrlCheck);
+		buttonGroup.add(jrbCrlCheckDistPoint);
+		buttonGroup.add(jrbCrlCheckFile);
 		buttonGroup.add(jrbOcspCheck);
 		buttonGroup.add(jrbChainCheck);
-		jrbCrlCheck.setSelected(true);
+		jrbCrlCheckDistPoint.setSelected(true);
 
-		Container pane = getContentPane();
-		pane.setLayout(new MigLayout("fill", "[right]unrel[]", "unrel[]unrel[]"));
-		pane.add(jlFormat, "");
-		pane.add(jrbCrlCheck, "split 3");
-		pane.add(jrbOcspCheck, "");
-		pane.add(jrbChainCheck, "wrap");
+		jcbSelectKeyStore = new JCheckBox(res.getString("DVerifyCertificate.jcbSelectKeyStore.text"));
 
-		pane.add(new JSeparator(), "spanx, growx, wrap");
-		
-		jcbSelectKeyStore  = new JCheckBox(res.getString("DVerifyCertificate.jcbSelectKeyStore.text"));
-		pane.add(jcbSelectKeyStore, "left, spanx, wrap");
-		
-		jcbSelectKeyStore.addItemListener(evt -> updateVerifyControls());
 		jcbKeyStore = new JComboBox<>(getKeystoreNames());
 		jcbKeyStore.setToolTipText(res.getString("DVerifyCertificate.jcbKeyStore.tooltip"));
-		jcbKeyStore.setPreferredSize(new Dimension(200,20));
+		jcbKeyStore.setPreferredSize(new Dimension(200, 20));
 		jcbKeyStore.setEnabled(false);
-		
+
 		jbLoadKeystore = new JButton();
 		jbLoadKeystore.setIcon(new ImageIcon(getClass().getResource("images/open.png")));
-		jbLoadKeystore.setToolTipText(res.getString("DVerifyCertificate.jbLoadKeystore.tooltip"));		
+		jbLoadKeystore.setToolTipText(res.getString("DVerifyCertificate.jbLoadKeystore.tooltip"));
 		jbLoadKeystore.setEnabled(false);
 
-		pane.add(new JLabel(res.getString("DVerifyCertificate.jlKeyStore.text")), "");
-		pane.add(jcbKeyStore, "split 2");
-		pane.add(jbLoadKeystore, "wrap");
-
 		jbOk = new JButton(res.getString("DVerifyCertificate.jbOk.text"));
 		jbCancel = new JButton(res.getString("DVerifyCertificate.jbCancel.text"));
-
 		jpButtons = PlatformUtil.createDialogButtonPanel(jbOk, jbCancel);
 
+		Container pane = getContentPane();
+		pane.setLayout(new MigLayout("fill", "[right]unrel[]", "unrel[]unrel[]"));
+		MiGUtil.addSeparator(pane, res.getString("DVerifyCertificate.jlCheckStatus.text"));
+		pane.add(jrbCrlCheckDistPoint, "wrap, left");
+		pane.add(jrbCrlCheckFile, "left");
+		pane.add(jtfCrlFile, "split 2");
+		pane.add(jbLoadCrl, "wrap");
+		pane.add(jrbOcspCheck, "wrap, left");
+		pane.add(new JSeparator(), "spanx, growx, wrap");
+		pane.add(jrbChainCheck, "wrap, left");
+		pane.add(new JSeparator(), "spanx, growx, wrap");
```

Comment:
I am unsure if this separator makes sense...

Reply:
![image](https://user-images.githubusercontent.com/58091322/122300910-fc36cf00-cecd-11eb-8f49-011503c9b147.png)

Would it be understood well?

Reply:
Well, I don't know, maybe "chain" is too short and has to be explained...

Reply:
How about this:

![grafik](https://user-images.githubusercontent.com/7840192/122690301-fcfe9680-d228-11eb-9722-9b0740996fad.png)

It is a lot of text, but it explains the different options pretty well I think.

Reply:
The modified resource bundle entries:
DVerifyCertificate.jlCheckStatus.text                       = Validate certificate chain and check revocation status using
DVerifyCertificate.jrbChainCheck.text                       = Do not check revocation status, only verify certificate chain
DVerifyCertificate.jcbSelectKeyStore.text                   = Use a different CA keystore for validating the certificate